app/views/idv/session_errors/failure.html.erb

```diff
@@ -16,6 +16,11 @@
           text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
+        IdentityConfig.store.in_person_proofing_enabled && {
+          url: idv_in_person_url,
+          text: t('in_person_proofing.headings.welcome'),
```

Comment:
Wondering if we should create a separate string for this, even if it's identical/duplicate text, so that we can update the heading of the Welcome screen independently from the link that's shown in the troubleshooting options. It seems like the sort of content change we might make and forget to consider the impact on the troubleshooting items, if shared.